corehq/apps/consumer_user/templates/change_password.html

```diff
@@ -0,0 +1,63 @@
+{% extends "consumer_user_base.html" %}
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load compress %}
+{% load static %}
+
+{% block title %}{% trans "Change Password" %}{% endblock title %}
+
+{% block background_content %}
+<div class="bg-container">
+  <div class="bg-full-cover-fixed bg-registration b-lazy b-loaded"
+        style="background-image: url({% static 'hqwebapp/images/molly.jpg' %});"></div>
+  <div class="bg-overlay"></div>
+</div>
+{% endblock %}
+
+{% block stylesheets %}
+<link type="text/css"
+      rel="stylesheet"
+      media="all"
+      href="{% static 'intl-tel-input/build/css/intlTelInput.css' %}" />
+
+{% compress css %}
+<link type="text/less"
+      rel="stylesheet"
+      media="all"
+      href="{% static 'registration/less/registration.less' %}" />
```

Comment:
this is causing the deploy to staging to fail, and I'm not sure why. 

```[10.201.10.72] Executing task 'compress'
[10.201.10.72] sudo: /home/cchq/www/staging/releases/2021-03-11_21.03/python_env-3.6/bin/python manage.py compress --force -v 0
[10.201.10.72] out: CommandError: An error occurred during rendering /home/cchq/www/staging/releases/2021-03-11_21.03/corehq/apps/consumer_user/templates/change_password.html: '/static/registration/less/registration.less' isn't accessible via COMPRESS_URL ('https://d2f60qxn5rwjxl.cloudfront.net/static/') and can't be compressed
```

Reply:
If you set `STATIC_CDN = 'https://d2f60qxn5rwjxl.cloudfront.net'` in your localsettings and run:
```
$ ./manage.py compress --force
```
you can repro this locally. Not sure if you can figure out where it is going wrong based on that.

Reply:
Fixed this compress issue